src/util.cpp

```diff
@@ -213,7 +213,13 @@ DHCPVal getDHCPValue(const config::Parser& config)
 
 bool getDHCPProp(const config::Parser& config, std::string_view key)
 {
-    return systemdParseLast(config, "DHCP", key, config::parseBool)
+    return systemdParseLast(config, "DHCPV4", key, config::parseBool)
+        .value_or(true);
+}
+
+bool getDHCP6Prop(const config::Parser& config, std::string_view key)
```

Comment:
Where is this getting called from?

Reply:
Its from the constructor of the Configuration where it checks for the value from config file, if not present then set the default value as true.